spk/shairport/src/installer.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin:${PATH}"
```

Comment:
Let's remove everything we don't need, which makes it look like this:
`PATH="${INSTALL_DIR}/bin:${PATH}"`